Presentation.Web/Controllers/API/V2/Internal/ItSystems/ItSystemLocalRegularOptionTypesInternalV2Controller.cs

```diff
@@ -16,20 +15,18 @@
 namespace Presentation.Web.Controllers.API.V2.Internal.ItSystems
 {
     [RoutePrefix("api/v2/internal/it-systems/{organizationUuid}/local-option-types")]
-    public class ItSystemLocalOptionTypesInternalV2Controller: InternalApiV2Controller
+    public class ItSystemLocalRegularOptionTypesInternalV2Controller: BaseLocalRegularOptionTypesInternalV2Controller<LocalBusinessType, ItSystem, BusinessType>
     {
-
-        private readonly IGenericLocalOptionsService<LocalBusinessType, ItSystem, BusinessType> _businessTypeService;
+        private readonly IGenericLocalOptionsService<LocalBusinessType, ItSystem, BusinessType> _localBusinessTypeService;
```

Comment:
you don't need this field, you can expose the injected one from the BaseController

Reply:
Unless it's not even needed here